tikv-configuration-file.md

```diff
@@ -819,6 +819,12 @@ Configuration items related to Raftstore.
 + Default value: `1024`
 + Minimum value: greater than `0`
 
+### `apply-yield-msg-size` <span class="version-mark">New in v6.4.0</span>
+
++ The maximum number of bytes the apply thread can handle for one FSM in one round of poll. This is a soft limit.
++ Default value: `"32KB"`
++ Minimum value: greater than `0`
```

Comment:
@glorv Please check if we need to use 32KB or 32KiB? 

And do we need to provide the Unit in the English version?

Reply:
done. Changed to 32KiB from @Oreoxmt 's suggestion.